Pod/Classes/Internal/SEGSegmentIntegration.m

```diff
@@ -528,6 +532,17 @@ - (void)applicationWillTerminate
     }];
 }
 
+-(void)applicationDidFinishLaunching
```

Comment:
Also, we *might* want to dispatch this to a background (serial) queue and call it. Reading / writing large files can take up quite a bit of time and we might be blockign the application from starting / UI from showing if we perform too much work here. (e.g. reading / writing 40mb files)